pkg/apiserver/experiment/experiment.go

```diff
@@ -70,8 +75,139 @@ func (s *Service) getExperimentDetail(c *gin.Context) {}
 // TODO: need to be implemented
 func (s *Service) deleteExperiment(c *gin.Context) {}
 
+// ExperimentInfo defines a form data of Experiment from API.
+type ExperimentInfo struct {
+	Name      string        `json:"name" binding:"required,NameValid"`
+	Namespace string        `json:"namespace" binding:"required,NameValid"`
+	Scope     ScopeInfo     `json:"scope"`
```

Comment:
What about add `Labels` and `Annotations` to let user fulfill the target podchao's labels and annotations.